src/frontend/apps/lti_site/apps/classroom/utils/parseDataElements/parseDataElements.ts

```diff
@@ -6,7 +6,6 @@ export const parseDataElements = (element: Element): ClassroomAppData => {
 
   if (context.modelName === ClassroomModelName.CLASSROOMS) {
     context.classroom = context.resource;
```

Comment:
If we only want to keep a generic way to access the resource ID, maybe we can do this ?
```suggestion
    context.classroom = context.resource;
    context.ressourceId = context.resource.id;
    delete context.resource;
```

Reply:
or even
```
context.jwtId = context.resource.id
```